expression/integration_test.go

```diff
@@ -2090,7 +2090,6 @@ func (s *testIntegrationSuite) TestBuiltin(c *C) {
 	c.Assert(charRecordSet, NotNil)
 	_, err = session.GetRows4Test(ctx, tk.Se, charRecordSet)
 	c.Assert(err.Error(), Equals, "unknown encoding: tidb")
-	c.Assert(rs.Close(), IsNil)
```

Comment:
Why remove this?

Reply:
It caused a double `Close`, which should be avoided.